.github/workflows/build.yaml

```diff
@@ -0,0 +1,41 @@
+# Checks that the build of the current branch succeeds
+
+name: build
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+    - 'src/**'
+  pull_request:
+    branches: [ main ]
+    paths:
+    - 'src/**'
+
+env:
+  DOTNET_VERSION: '6.0.x'   # The .NET SDK version to use
+  CONFIGURATION: 'Release'  # The configuration to use
+
+jobs:
+  build:
+    name: build-${{matrix.os}}
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: ${{ env.DOTNET_VERSION }}
+
+    - name: Install dependencies
+      run: dotnet restore ./src
+
+    - name: Build
+      run: dotnet build ./src --configuration ${{ env.CONFIGURATION }} --no-restore
```

Comment:
How did these get in again? 😅 
Try to pull the parser branch from this origin.

Reply:
Very, very good question. Git just did not want to cooperate.